File: module-3-java/bi-tool/pages/components/user_transfer.js
import MUIDataTable from 'mui-datatables'

const columns = [
	{
		name: 'sender',
		label: 'Sender',
		options: {
			filter: false,
			sort: false
		}
	},
	{
		name: 'receiver',
		label: 'Receiver',
		options: {
			filter: false,
			sort: false
		}
	},
	{
		name: 'transId',
		label: 'Transaction ID',
		options: {
			filter: false,
			sort: false
		}
	},
	{
		name: 'reqDate',
		label: 'Request Date',
		options: {
			filter: false,
			sort: false
		}
	},
	{
		name: 'amount',
		label: 'Amount',
		options: {
			sort: true
		}
	}
]

const options = {
	download: false,
	print: false
}

const UserTransfer = ({ data }) => {
	return (
		<MUIDataTable title={'User Transfer'} data={data} columns={columns} options={options} className="w-2/3 my-2" />
	)
}

export default UserTransfer

File: module-3-java/spring-boot-tut/src/main/java/com/example/demo/model/UserTransfer.java
package com.example.demo.model;

import java.util.Date;

import javax.persistence.*;

import lombok.Data;

/**
 * The type User transfer.
 */
@Entity
@Data
@Table(name = "user_transfer")
public class UserTransfer {
	/**
	 * Auto Generated ID.
	 */
	@Id
	@GeneratedValue
	private String id;

	/**
	 * Sender ID.
	 */

	@Column(name = "sender")
	private String sender;

	/**
	 * Receiver ID.
	 */

	@Column(name = "receiver")
	private String receiver;

	/**
	 * Transfer ID.
	 */

	@Column(name = "trans_id")
	private String transId;

	/**
	 * Date Transfer.
	 */

	@Column(name = "req_date")
	private Date reqDate;

	/**
	 * Amount of Transfer Money.
	 */

	@Column(name = "amount")
	private double amount;

}
